packages/slider/src/Slider.ts

```diff
@@ -223,7 +223,7 @@ export class Slider extends Focusable {
         return html`
             <div class="ticks">
                 ${ticks.map(
-                    (tick, i) => html`
+                    (_, i) => html`
```

Comment:
You could name this _tick to still convey intent. Ditto for above 'size' change (_size)

Reply:
Cool that it works both ways. I'd not used this feature in TS before, but now that we've got this on it's good to know my options. That's for the support in getting this addressed